test/conftest.py

```diff
@@ -41,43 +42,36 @@ def test_dir() -> Path:
 
 
 @pytest.fixture(scope="session")
-def pudl_settings_fixture() -> Any:  # noqa: C901
+def pudl_input_dir() -> dict[Any, Any]:
+    """Determine where the PUDL input/output dirs should be."""
+    input_override = None
+
+    # In CI we want a hard-coded path for input caching purposes:
+    if os.environ.get("GITHUB_ACTIONS", False):
+        # hard-code input dir for CI caching
+        input_override = Path(os.environ["HOME"]) / "pudl-work/data"
+
+    return {"input_dir": input_override}
```

Comment:
What all do you expect to find in this directory? It looks like maybe it should be pointing at `$PUDL_INPUT` in the new Dagsterized PUDL?  Is this just cribbed from the re-vamped PUDL test environment?

Reply:
Ya I took this from the PUDL test configuration. `pudl_work/data` is set as the `$PUDL_INPUT` location in CI, so maybe this is redundant?